prow/plugins/label/label.go

```diff
@@ -36,7 +37,18 @@ var (
 )
 
 func init() {
-	plugins.RegisterGenericCommentHandler(pluginName, handleGenericComment, nil)
+	plugins.RegisterGenericCommentHandler(pluginName, handleGenericComment, helpProvider)
+}
+
+func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	// The Config field is omitted because this plugin is not configurable.
+	return &pluginhelp.PluginHelp{
+			Description: "The label plugin provides commands that add or remove certain types of labels. Labels of the following types can be manipulated: 'area/*', 'priority/*', 'kind/*', and 'sig/*'.",
+			WhoCanUse:   "Only members of the repository's organization may use the label commands.",
+			Usage:       "/(remove-)(area|priority|kind|sig) <target>",
```

Comment:
Sometimes we use brackets to denote optional fields in one these, sometimes parentheses -- does it matter? Should we choose one for consistency?

Reply:
I was following http://docopt.org/ which does make a distinction. `[]` mean optional while `()` is for grouping and still indicates required args.
This should be `/[remove-](area|priority|kind|sig) <target>`.